src/ReactionService.Models.Dto/Configurations/RabbitMqConfig.cs

```diff
@@ -1,11 +1,16 @@
 ﻿using LT.DigitalOffice.Kernel.BrokerSupport.Attributes;
 using LT.DigitalOffice.Kernel.BrokerSupport.Configurations;
-using LT.DigitalOffice.Models.Broker.Requests.User;
+using LT.DigitalOffice.Models.Broker.Requests.Image;
 
 namespace LT.DigitalOffice.ReactionService.Models.Dto.Configurations;
 
 public class RabbitMqConfig : BaseRabbitMqConfig
 {
-  [AutoInjectRequest(typeof(IGetUsersDataRequest))]
-  public string GetUsersDataEndpoint { get; set; }
+  // Image
+
+  [AutoInjectRequest(typeof(IGetImagesRequest))]
+  public string GetImagesEndpoint { get; set; }
+
+  [AutoInjectRequest(typeof(ICreateImagesRequest))]
+  public string CreateImagesEndpoint { get; set; }
```

Comment:
usually add endpoints when start using them. it's not mistake, let it be, but then make sure you use them